tests/test_postblock.py

```diff
@@ -3,44 +3,170 @@
 -------------------------------------------------------
 Content:
 """
+import pytest
+import yaml
+import os
+import logging
 
 import torch
-from credit.postblock import PostBlock
-from credit.postblock import (
-    SKEBS,
-    TracerFixer,
-    GlobalMassFixer,
-    GlobalWaterFixer,
-    GlobalEnergyFixer,
-)
-
-
+import torch.nn as nn
+from credit.models.crossformer import CrossFormer
+from credit.postblock import GlobalWaterFixer, PostBlock, Backscatter_FCNN
+from credit.postblock import SKEBS, TracerFixer, GlobalMassFixer, GlobalEnergyFixer
+from credit.parser import credit_main_parser
+from credit.postblock import Backscatter_CNN
+
+
+TEST_FILE_DIR = "/".join(os.path.abspath(__file__).split("/")[:-1])
+CONFIG_FILE_DIR = os.path.join("/".join(os.path.abspath(__file__).split("/")[:-2]),
+                      "config")
+
+@pytest.mark.skip(reason="need to have model weights, to test")
```

Comment:
How big are the skebs model weights? Is there a way to make a small test version for this?